### PR TITLE
docs: add manual installation instructions

### PR DESCRIPTION
**Pull request by ffrosch**
Added a few lines to indicate how to install this extension.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,18 @@ A context menu entry for Dolphin to open VS Code at the current location.
...
-
 ## Download
 
 <a href="https://store.kde.org/p/1413799/"><img alt="Download from the KDE Store" height="50px"
 src="https://img.shields.io/badge/KDE_Store-blue?style=for-the-badge&logo=kde"></a>
 <a href="https://github.com/Merrit/kde-dolphin-open-vscode/releases/latest"><img alt="Download from GitHub" height="50px"
 src="https://img.shields.io/badge/GitHub-blue?style=for-the-badge&logo=github"></a>
 
+## Install
+
```
Comment (Merrit) [inline README.md]: Should probably clarify this is only necessary if not downloading the automatic way through the store.

```suggestion
## Manual Install

If not downloading from the KDE Store, install manually:

```

File: README.md [hunk 2]
```diff
@@ -4,14 +4,18 @@ A context menu entry for Dolphin to open VS Code at the current location.
...
+Copy the file `openVSCode.desktop` into the `ServiceMenus` folder:
+1. Create folders: `mkdir -p ~/.local/share/kservices5/ServiceMenus`
+2. Move file: `mv openVSCode.desktop ~/.local/share/kservices5/ServiceMenus`
```
Comment (Merrit) [inline README.md]: This is no longer the path for Plasma 6, for users on newer releases the path will have to be `~/.local/share/kio/servicemenus/`

**Comment by Merrit** [comment]
Thanks for the PR! Just a couple suggestions :)

File: README.md (hunk 1 above)
Comment (ffrosch) [reply README.md]: True, thanks for the attention to detail @Merrit  :-)

File: README.md (hunk 2 above)
Comment (ffrosch) [reply README.md]: Thanks, I'll adjust it!

**Review by ffrosch — COMMENTED** [review]
Integrate clarification on type of install

**Comment by Merrit** [comment]
LGTM. Thanks for your contribution! :tada: :blue_heart:

**Pull request merged**